hub/api/dataset.py

```diff
@@ -262,6 +262,39 @@ def load(
         except AgreementError as e:
             raise e from None
 
+    @staticmethod
+    def rename(
```

Comment:
path and name are confusing. Original path and new path or original name and new name might be better